docs/IWYUMappings.md

```diff
@@ -102,12 +97,11 @@ For example;
 
 The rationale for the `ref` directive was to make it easier to compose project-specific mappings from a set of library-oriented mapping files. For example, IWYU might ship with mapping files for [Boost](http://www.boost.org), the SCL, various C standard libraries, the Windows API, the [Poco Library](http://pocoproject.org), etc. Depending on what your specific project uses, you could easily create an aggregate mapping file with refs to the relevant mappings.
 
-
-#### Specifying Mapping Files ####
+### Specifying Mapping Files ###
 
 Mapping files are specified on the command-line using the `--mapping_file` switch:
 
-    $ include-what-you-use -Xiwyu --mapping_file=foo.imp some_file.cc
```

Comment:
I like the `$` as a command-line prompt indication, can we please keep that?

Reply:
Done. Added an exception for this rule.